src/client/lobby/room-creator.spec.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import RoomCreator from './room-creator';
+
+describe('<RoomCreator />', () => {
+    beforeEach(() => {
+        window.cloak = jasmine.createSpyObj('cloak', ['configure', 'run']);
```

Comment:
This isn't required  -> it was used when cloak needed to be mocked out for a component, which isn't right here